src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj

```diff
@@ -12,4 +12,7 @@
     <Commandlineparameters></Commandlineparameters>
     <RunWithWarnings>true</RunWithWarnings>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />

```

Comment:
maybe we should use `System.Json`?

Reply:
That's a good one! 
I thought that was .NET Core. It's net standard :)